src/pages/EventDetails/components/EventRenderer.tsx

```diff
@@ -289,7 +289,7 @@ const EventRenderer = ({ data, preview = false }: EventRendererProps) => {
               Dette arrangementet gir ikke prikker
             </Alert>
           )}
-          {Boolean(data.registration_priorities.length) && data.registration_priorities.length !== 14 && (
+          {Boolean(data.registration_priorities.length) && (
```

Comment:
Genuint spørsmål fra min side fordi jeg ikke vet: Hvorfor fjerner du `data.registration_priorities.length !== 14 && ` her i stedet for å oppdatere lengden?

Reply:
Så nå kommentaren til Olaf lenger opp og fikk svar på spørsmålet mitt :)

Reply:
Fordi det er dumt å ha en konstant som skaper problemer hvis det legges til/fjernes flere studier eller klasser. I tillegg kan vi like gjerne vise alle prioterte selv som alle er prioritert.